README.md

```diff
@@ -50,28 +50,33 @@ rm -rf /var/lib/microshift && rm -r $HOME/.microshift
 
 You can locally build Microshift using one of two methods, either using a container build (recommended) on Podman or Docker:
 ```
+sudo yum -y install make golang
 make microshift
 ```
 
-or directly on the host after installing the build-time dependencies
+or directly on the host after installing the build-time dependencies. When using RHEL ensure the system is registered and run the following before installing the prerequisites.
+
+```
+ARCH=$( /bin/arch )
+sudo subscription-manager repos --enable "codeready-builder-for-rhel-8-${ARCH}-rpms"
+```
+
+The following packages are required for Fedora and RHEL.
 ```
-sudo dnf install -y glibc-static
+sudo yum install -y glibc-static gcc make golang
 make
 ```
 
 ### Environment Configuration
 
 Before running Microshift, the host must first be configured.  This can be handled by running  
 
-`CONFIG_ENV_ONLY=true ./install.sh`
-
-### Running
-
 ```
-sudo microshift run
+CONFIG_ENV_ONLY=true ./install.sh
 ```
 
 Microshift keeps all its state in its data-dir, which defaults to `/var/lib/microshift` when running Microshift as privileged user and `$HOME/.microshift` otherwise. Note that running Microshift unprivileged only works without node role at the moment (i.e. using `--roles=controlplane` instead of the default of `--roles=controlplane,node`).
 
-You can find the kubeadmin's kubeconfig under `$DATADIR/resources/kubeadmin/kubeconfig`.
+### Kubeconfig
+The `install.sh` script should place the kubeconfig file for you. If you need it for another user or to use externally the kubeadmin's kubeconfig is placed `/var/lib/microshift/resources/kubeadmin/kubeconfig` during configuration.
```

Comment:
To be precise, it's placed there the first time `microshift run` is executed. IOW, after running `install.sh` with `CONFIG_ENV_ONLY` flag on a fresh dev env, kubeconfig will not be available in either location.

Reply:
That is correct. 

CONFIG_ENV_ONLY=true env var was meant for CI to basically configure the dependencies on the host, but not to deploy microshift itself.

The kubeconfig files are generated once microshift runs, and CONFIG_ENV_ONLY stops right before fetching microshift and preparing the `.kube/config` file.